### PR TITLE
fix: deprecation of root() called by name

### PR DESCRIPTION
**Pull request by luca-simonetti**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "limenius/react-bundle",
```
Comment (nacmartin) [inline composer.json:2]: ???????

Comment (nacmartin) [reply composer.json:2]: Is there an explanation for this last commit of the PR?

**Comment by luca-simonetti** [comment]
sorry. I made my own packagist version, so I had committed that. Totally forgot about this PR. I'm closing it now since I don't need it anymore to be merged.

**Comment by nacmartin** [comment]
Ah, ok that is the explanation then :D 

Sorry, didn't have time to check this out, I am glad you did this PR so I know for sure what needs to be done (but I have to update the sandbox also to sf5).

**Pull request closed without merging**